browser/js/dashboard/gridbox/dashboard.factory.js

```diff
@@ -53,17 +61,40 @@ app.factory('DashboardFactory', function($http, $q, GeneratorFactory){
     			refreshInterval: e.refreshInterval || 1000,
     			type: e.type,
     			sizeX: e.sizeX,
-    			sizeY: e.sizeY, 
+    			sizeY: e.sizeY,
     			col: e.col,
     			row: e.row,
                 color: e.color || '#0000ff',
-                xparam: e.xparam, 
+                xparam: e.xparam,
     			dashboardId: dashboardId,
                 yparam: e.yparam
     		}
     	})
     	return $http.put('/api/dashboards/' + dashboardId, thing)
```

Comment:
Probably have told you about them, but es6 template strings can be a nice alternative for simple string concatenation. For example, the above could instead be `` `/api/dashboards/${dashboardId}` ``.